bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -3874,6 +3874,11 @@
         <sqlFile path="V1_98_WardsListSql.sql"/>
     </changeSet>
 
+    <changeSet id="bahmni-PatientSearch-Update-202201250315" author="Kavitha S, Soorya Kumaran C">
```

Comment:
can you please use 3 different changesets as suggested above? also please make sure that the changeset clearly says which is for which property

Reply:
Okay